go-controller/pkg/cni/bandwidth.go

```diff
@@ -87,11 +89,17 @@ func getPodBandwidth(ifname string) (ingressBPS, egressBPS int64, err error) {
 	}
 	// egressBPS
 	out, err = ovsGet("interface", ifname, "ingress_policing_rate", "")
-	if err == nil && len(out) > 0 {
-		egressBPS, err = strconv.ParseInt(out, 10, 64)
-		if err == nil {
-			egressBPS *= 1000
+	if err != nil {
+		return 0, 0, errors.Wrapf(err, "failed to get ingress_policing_rate for interface %s", ifname)
+	}
+	if len(out) > 0 {
+		egressValue, err := strconv.ParseInt(out, 10, 64)
+		if err != nil {
+			return 0, 0, errors.Wrapf(err, "failed to parse ingress_policing_rate %s", out)
+		}
+		if egressValue != 0 { // 0 is the default value so we return as it was not found
```

Comment:
why even check this?

Reply:
Because we want to fallback returning -1 if the value is 0, so the check here https://github.com/ovn-org/ovn-kubernetes/blob/e0ab509b5dd549168adc358fbb74ec0dfe118424/go-controller/pkg/cni/cni.go#L179 won't fail

Reply:
let's change the code to just use error to signal not found, and always return 0 or the value. Let's stop using negative 1.